addons/hr_skills_slides/__manifest__.py

```diff
@@ -18,6 +18,7 @@
         'views/hr_employee_views.xml',
         'views/hr_templates.xml',
         'data/hr_resume_data.xml',
+        'data/mail_templates_chatter.xml',
```

Comment:
can we add a test with `assertQueryCount` in a commit before this one, and update the count in this commit to see the improvement ?